model/patch/patch.go

```diff
@@ -678,7 +679,7 @@ func (p *Patch) UpdateGithashProjectAndTasks() error {
 		"$set": bson.M{
 			GithashKey:       p.Githash,
 			PatchesKey:       p.Patches,
-			PatchedConfigKey: p.PatchedConfig,
+			PatchedConfigKey: p.PatchedParserProject,
```

Comment:
This is called after the patch is updated to refresh a patch's config when a commit queue job runs to save the altered properties to the DB. Since patchedProjectConfig is a part of these props that get modified we do need to add it. I've done so.